package.json

```diff
@@ -116,6 +116,7 @@
     "tvm": "^0.8.14",
     "typedoc": "https://github.com/coveord/typedoc",
     "typescript": "^2.8.1",
+    "underscore": "1.8.x",
```

Comment:
I tried upgrading underscore to 1.9 because I thought I would need a new feature but the new version has some regressions and turns out I don't need the new feature. When reverting to the old version I used this form on semver. I'll switch it back.